src/software/ai/hl/stp/play/dribbling_parcour_play_test.cpp

```diff
@@ -0,0 +1,69 @@
+#include "software/ai/hl/stp/play/dribbling_parcour_play.h"
+
+#include <gtest/gtest.h>
+
+#include "software/simulated_tests/simulated_play_test_fixture.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/world/world.h"
+
+class DribblingParcourPlayTest : public SimulatedPlayTestFixture
+{
+   protected:
+    Field field = Field::createSSLDivisionBField();
+};
+
+TEST_F(DribblingParcourPlayTest, test_dribbling_parcour_play_stopped)
+{
+    BallState ball_state(Point(-0.8, 0), Vector(0, 0));
+    auto friendly_robots = TestUtil::createStationaryRobotStatesWithId(
+        {Point(4, 0), Point(-2, -1), Point(-3, 1)});
```

Comment:
Why do we need 3 friendly robots? Shouldn't we set up the enemy robots here?

Reply:
good point, removed a friendly robot. Setting up all the enemy robots is part of the dribbling parcour ticket, this test is just to check that the gamestate transition is working